guides/training_keras_models_on_cloud.py

```diff
@@ -1,421 +1,303 @@
 """
 Title: Training Keras models with TensorFlow Cloud
-Author: [Jonah Kohn](https://jonahkohn.com)
+Authors: [Jonah Kohn](https://jonahkohn.com), [Sina Chavoshi](https://www.linkedin.com/in/sinachavoshi/)
 Date created: 2020/08/11
-Last modified: 2020/08/11
-Description: In-depth usage guide for TensorFlow Cloud.
+Last modified: 2021/07/23
+Description: Usage guide for TensorFlow Cloud.
 """
+
 """
 ## Introduction
 
-[TensorFlow Cloud](https://github.com/tensorflow/cloud) is a Python package that
-provides APIs for a seamless transition from local debugging to distributed training
-in Google Cloud. It simplifies the process of training TensorFlow models on the
-cloud into a single, simple function call, requiring minimal setup and no changes
-to your model. TensorFlow Cloud handles cloud-specific tasks such as creating VM
-instances and distribution strategies for your models automatically. This guide
-will demonstrate how to interface with Google Cloud through TensorFlow Cloud,
-and the wide range of functionality provided within TensorFlow Cloud. We'll start
-with the simplest use-case.
-"""
+TensorFlow Cloud is a library that makes it easier to do training and
+hyperparameter tuning of Keras models on Google Cloud.
 
-"""
-## Setup
+Using TensorFlow Cloud's `run` API, you can send your model code directly to
+your Google Cloud account, and use Google Cloud compute resources without
+needing to login and interact with the Cloud UI (once you have set up your
+project in the console).
 
-We'll get started by installing TensorFlow Cloud, and importing the packages we
-will need in this guide.
-"""
+This means that you can use your Google Cloud compute resources from inside
+directly a Python notebook: a notebook just like this one! You can also send
+models to Google Cloud from a plain `.py` Python script.
 
-"""shell
-pip install -q tensorflow_cloud
-"""
+## Simple example
 
-import tensorflow as tf
-import tensorflow_cloud as tfc
+This is a simple introductory example to demonstrate how to train a model
+remotely using [TensorFlow Cloud](https://tensorflow.org/cloud) and Google
+Cloud.
 
-from tensorflow import keras
-from tensorflow.keras import layers
+You can just read through it to get an idea of how this works, or you can run
+the notebook in Google Colab. Running the notebook requires connecting to a
+Google Cloud account and entering your credentials and project ID. See [Setting
```

Comment:
Links should be on a single line (even if it's long) to ensure proper rendering.

Reply:
Done.